.vscode/settings.json

```diff
@@ -3,5 +3,8 @@
         "tests"
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
```

Comment:
Il ne faut pas commiter le settings.json

Reply:
reverted to previous version